ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -285,15 +285,15 @@ export const Homepage = fp.flow(
       }
     }
 
-    async checkWorkspaces() {
+    async loadWorkspaces() {
```

Comment:
just a suggestion, rename this to fetchWorkspaces, to differentiate the behavior from the loadWorkspaces function in recent-workspaces